docs/userguide.md

```diff
@@ -7,8 +7,18 @@ is a walkthrough tutorial for how to deploy the federation v2 control plane.
 
 ### Binaries
 
-The federation deployment depends on `apiserver-boot`, `crinit`, and `kubectl`
-being installed in the path. These binaries can be installed via the
+Federation can be deployed with a prebuilt image from federation-v2 master using `scripts/deploy-federation-latest.sh`.
+The federation deployment depends on [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) being installed in the path.
+
+Alternatively, you can build your own image with `scripts/deploy-federation.sh <image> cluster2 cluster3`
```

Comment:
This is more like install, can we delete this and refer to following section for build?